pkg/storage/utils/ace/ace.go

```diff
@@ -365,6 +368,9 @@ func getACEPerm(set *provider.ResourcePermissions) string {
 	if set.RemoveGrant || set.UpdateGrant {
 		b.WriteString("o")
 	}
+	if set.DenyGrant {
+		b.WriteString("O")
```

Comment:
Why `"O"`? We are using linux acls here: https://linux.die.net/man/5/nfs4_acl There is no capital `"O"` flag afais.

Reply:
small `o` is "change owner"

There is nothing there for "revoke acess" so i picked capital "O". Open for suggestions.